README.md

```diff
@@ -17,6 +17,14 @@ npm install
 npm run build
 ```
 
+If you are looking for a .zip version of the plugin to install on your site use this link:
```

Comment:
I suggest simplifying that to: `You can also download a [packaged .zip version](https://sitekit.withgoogle.com/news/site-kit-developer-preview/) of the plugin to install on your site.`

Reply:
Sounds good to me. As long as people have a way to easily obtain a packaged .zip version this is fine. Feel free to change this and merge it.